.env.example

```diff
@@ -0,0 +1,3 @@
+MYSQL_DATABASE=school
+MYSQL_USER=internal
+MYSQL_PASSWORD=secret
```

Comment:
Zlý example súbor.. tu má byť iné heslo ktoré sa používa v kóde.
Buď zmeniť heslo tu, alebo php apku prinútiť používať .env súbor